src/HOHQMesh.jl

```diff
@@ -22,8 +22,12 @@ end
 # Main wrapper to generate a mesh from a control file
 export generate_mesh
 
+# Make the examples directory available to the user
+export examples_dir
```

Comment:
Won't this clash with Trixi's `examples_dir` export?

Reply:
probably. but from a user point of view it is incredibly annoying to always type `HOHQMesh.examples_dir()`. Should I just pick a different name or something?

Reply:
TBH, I dunno. @ranocha Do you have a good suggestions for this?

Reply:
It will clash but I don't see a way around this. If people are `using Trixi` and `using HOHQMesh` they will get notified. If they are `using` only one of them, everything is fine. I would leave it as it is.

Reply:
Okay, I will leave it for now but add it to the meta-issue list